types/index.d.ts

```diff
@@ -4,7 +4,16 @@ import * as Unist from 'unist'
 import * as vfileMessage from 'vfile-message'
 
 declare namespace vfile {
-  type VFileContents = string | Buffer
+  /**
+   * Type must have the same base as Node.js Buffer
+   */
+  type BufferLike = Uint8Array
+  /**
+   * Type must have same base as Node.js BufferEncoding
+   */
+  type BufferEncodingLike = string
```

Comment:
the exact encoding literals supported change overtime, it is documented to be a string
https://nodejs.org/api/buffer.html#buffer_class_method_buffer_isencoding_encoding

Reply:
Same here, I think the `toString` method might as well be defined like this,with the `BufferEncodingLike` type omitted.

```
toString: (encoding?: string) => string
```

Reply:
updated in 14c3c95